.github/workflows/behat-test.yml

```
@@ -125,7 +125,11 @@ jobs:
 
       - name: Restore standard slim image from cache
         id: cache-docker-slim
+<<<<<<< HEAD
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 128: could not find expected ':'

(syntax-check)